### PR TITLE
Update Signing Links

### PR DESCRIPTION
**Pull request by princespaghetti**
docs/website: update link to signing section
cmd: Update help text to link to signing format section

Fixes: #3027
Signed-off-by: Anthony Barbieri <anthonyabarbieri@gmail.com>

### CODE REVIEW

File: cmd/flags.go [hunk 1]
```diff
@@ -91,7 +91,7 @@ func addSigningAlgFlag(fs *pflag.FlagSet, alg *string, value string) {
 }
 
 func addClaimsFileFlag(fs *pflag.FlagSet, file *string) {
-	fs.StringVarP(file, "claims-file", "", "", "set path of JSON file containing optional claims (see: https://openpolicyagent.org/docs/latest/management/#bundle-signature-format)")
+	fs.StringVarP(file, "claims-file", "", "", "set path of JSON file containing optional claims (see:  https://openpolicyagent.org/docs/latest/management/#signature-format)")
```
Comment (srenatus) [inline cmd/flags.go:94]: [nit] Can you drop the extra space, by any chance? (I don't suppose that it's a display issue with github only; if so, ignore me 😄)

**Review by srenatus — COMMENTED** [review]
Thanks a lot for fixing this! One nitpick inside, otherwise LGTM 🚀

File: cmd/flags.go (hunk 1 above)
Comment (princespaghetti) [reply cmd/flags.go]: Ah, yep sorry about that, just pushed up the fix 😄

File: cmd/build.go [hunk 2]
```diff
@@ -157,7 +157,7 @@ To include additional claims in the payload use the --claims-file flag to provid
 containing optional claims.
 
 For more information on the format of the ".signatures.json" file
-see https://www.openpolicyagent.org/docs/latest/management/#bundle-signature.
+see https://openpolicyagent.org/docs/latest/management/#signature-format.
```
Comment (anderseknert) [inline cmd/build.go:160]: The www sub-domain should be kept here.

**Review by anderseknert — CHANGES_REQUESTED** [review]

**Comment by anderseknert** [comment]
Just a minor comment, but looks good @princespaghetti 👍 This only fixes half of the issue though, as even the correct link does not currently describe "the purpose of adding extra claims and why one might want those", as expected from the linked issue. This doesn't block merging as this is still an improvement, but I'm not sure it should close the issue. Since I don't really understand the purpose of this flag myself, perhaps someone else can provide that.

**Comment by ashutosh-narkar** [comment]
The purpose of the `--claims-file` flag is to allow inclusion of additional claims in the payload of the bundle signature. The idea here is that we may want to perform verification based on these claims in the future but for now they will be only be used for informational purposes ie,  unused for verification.

> Since I don't really understand the purpose of this flag myself, perhaps someone else can provide that.

**Comment by anderseknert** [comment]
Thanks @ashutosh-narkar - should I add that to the docs or will you do it?

**Comment by ashutosh-narkar** [comment]
> Thanks @ashutosh-narkar - should I add that to the docs or will you do it?

Either way works. We can also make it part of this PR by adding a note (block quotes) about this in the bundle signing section.

File: cmd/build.go (hunk 2 above)
Comment (princespaghetti) [reply cmd/build.go]: added back in!

**Comment by tsandall** [comment]
Can we merge this now?

**Pull request merged**